rosbag2_compression/test/rosbag2_compression/fake_compressor.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <string>
+
+#include "fake_compressor.hpp"
```

Comment:
extremely nitpick, but we could consider renaming this to `MockCompressor` to stay in line with the other rosbag2 tests

Reply:
Isn't it technically a fake instead of a mock since we aren't mocking the base interface, rather we're creating a fake implementation?

I just want to make sure we don't confuse ourselves (or someone else) down the line.